apps/api-extractor/src/api/IConfigFile.ts

```diff
@@ -405,4 +406,15 @@ export interface IConfigFile {
    * to prevent spurious diffs in output files tracked for tests.
    */
   testMode?: boolean;
+
+  /**
+   * Specifies how API Extractor sorts members of an enum when generating api.json, api.md and rollup.d.ts.
+   * The default behaviour has been alphabetical, ByName in this case.
+   *
+   * - "by-name": sort the items according to the enum member name
+   * - "preserve": keep the original order that items appear in the source code
```

Comment:
API documenter won't render this doc comment as a nice Markdown list on a documentation site because TSDoc actually doesn't parse Markdown lists from doc comment. So maybe consider following the doc comment style used in `newlineKind` above, and keep this as a paragraph. In general, consider also adding this information to a `@remarks` tag (following the current style in the file).